{"identifier":{"url":"doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError\/Error-Implementations","interfaceLanguage":"swift"},"sections":[],"schemaVersion":{"major":0,"patch":0,"minor":3},"metadata":{"title":"Error Implementations","role":"collectionGroup","roleHeading":"API Collection","modules":[{"name":"Arachne"}]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/arachne\/arerror\/error-implementations"]}],"hierarchy":{"paths":[["doc:\/\/arachne.Arachne\/documentation\/Arachne","doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError"]]},"kind":"article","topicSections":[{"anchor":"Instance-Properties","identifiers":["doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError\/downloadResumeData","doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError\/localizedDescription"],"generated":true,"title":"Instance Properties"}],"references":{"doc://arachne.Arachne/documentation/Arachne/ARError/downloadResumeData":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"downloadResumeData"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Data","preciseIdentifier":"s:10Foundation4DataV"},{"kind":"text","text":"?"}],"type":"topic","identifier":"doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError\/downloadResumeData","title":"downloadResumeData","url":"\/documentation\/arachne\/arerror\/downloadresumedata","role":"symbol","abstract":[{"text":"Returns data useful to resume a failed download, if any","type":"text"}],"kind":"symbol"},"doc://arachne.Arachne/documentation/Arachne":{"identifier":"doc:\/\/arachne.Arachne\/documentation\/Arachne","role":"collection","abstract":[{"text":"Arachne is a lightweight, minimalistic, zero dependencies networking layer for apps using Swift Concurrency (async\/await) developed in Swift, that provides an opinionated abstraction layer to remove boilerplate code.","type":"text"}],"kind":"symbol","type":"topic","url":"\/documentation\/arachne","title":"Arachne"},"doc://arachne.Arachne/documentation/Arachne/ARError/localizedDescription":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}],"role":"symbol","kind":"symbol","url":"\/documentation\/arachne\/arerror\/localizeddescription","title":"localizedDescription","type":"topic","identifier":"doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError\/localizedDescription","abstract":[]},"doc://arachne.Arachne/documentation/Arachne/ARError":{"abstract":[{"type":"text","text":"Errors that can be thrown by Arachne networking library"}],"type":"topic","title":"ARError","kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ARError","kind":"identifier"}],"identifier":"doc:\/\/arachne.Arachne\/documentation\/Arachne\/ARError","url":"\/documentation\/arachne\/arerror","role":"symbol","navigatorTitle":[{"text":"ARError","kind":"identifier"}]}}}